src/averageAges.js

```diff
@@ -20,6 +20,25 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const menOfPeople = people.filter(x => x.sex === 'm');
+
+  const callback = (total, next) => {
+    return total + (next.died - next.born);
+  };
+
+  if (century !== undefined) {
```

Comment:
As described in the task description you should try to avoid using `if` statement and rather use a ternary operator.

`const menOfPeopleCentury = century !== undefined ? menOfPeople.filter(x => Math.ceil(x.died / 100) === century) : [];`
`const ageSumCentury = menOfPeopleCentury.reduce(callback, 0);`
`const averageAgeCentury = menOfPeopleCentury.length > 0 ? ageSumCentury / menOfPeopleCentury.length : 0;`

`return averageAgeCentury;`

try using something like this also for the `calculateWomenAverageAge` and `calculateAverageAgeDiff` try refactor them to not using `if` statement